tools/wordcloud/wordcloud.xml

```diff
@@ -0,0 +1,210 @@
+        <param argument="--include_numbers" type="boolean" truevalue="True" falsevalue="False" value="False" label="Whether to include numbers as phrases or not"/>
+        <param argument="--min_word_length" value="0" type="integer" label="Minimum number of letters a word must have to be included"/>
+        <param argument="--prefer_horizontal" value="0.9" type="float" label="Ratio of times to try horizontal fitting as opposed to vertical" min="0" max="1"/>
+        <param argument="--scale" value="1.0" type="float" label="Scaling between computation and drawing"/>
+        <param argument="--colormap" type="select" label="Matplotlib colormap name" value="viridis">
+            <option value="viridis">viridis</option>
+            <option value="plasma">plasma</option>
+            <option value="inferno">inferno</option>
+            <option value="magma">magma</option>
+            <option value="cividis">cividis</option>
+            <option value="Greys">Greys</option>
+            <option value="Purples">Purples</option>
+            <option value="Blues">Blues</option>
+            <option value="Greens">Greens</option>
+            <option value="Oranges">Oranges</option>
+            <option value="Reds">Reds</option>
+            <option value="YlOrBr">YlOrBr</option>
+            <option value="YlOrRd">YlOrRd</option>
+            <option value="OrRd">OrRd</option>
+            <option value="PuRd">PuRd</option>
+            <option value="RdPu">RdPu</option>
+            <option value="BuPu">BuPu</option>
+            <option value="GnBu">GnBu</option>
+            <option value="PuBu">PuBu</option>
+            <option value="YlGnBu">YlGnBu</option>
+            <option value="PuBuGn">PuBuGn</option>
+            <option value="BuGn">BuGn</option>
+            <option value="YlGn">YlGn</option>
+        </param>
+        <param argument="--mode" type="select" label="Use RGB or RGBA for transparent background" value="RGB">
+            <option value="RGB">RGB</option>
+            <option value="RGBA">RGBA</option>
+        </param>
+        <param argument="--max_words" value="200" type="integer" label="Maximum number of words"/>
+        <param argument="--width" value="400" type="integer" label="Define output image width"/>
+        <param argument="--height" type="integer" value="200" label="Define output image height"/>
+        <param argument="--random_state" type="integer" label="Random seed"/>
+        <param argument="--no_normalize_plurals" value="True" type="boolean" optional="True" label="Whether to remove trailing s from words"/>
+        <param argument="--repeat" type="boolean" truevalue="True" falsevalue="False" value="False" label="Whether to repeat words and phrases until max_words or min_font_size is reached"/>
+        <param argument="--version" type="boolean" truevalue="True" falsevalue="False" value="False" label="Show program's version number"/>
```

Comment:
version should not be offered to the user, but you should add a `<version_command>` tag to the tool.

Reply:
> version should not be offered to the user, but you should add a `<version_command>` tag to the tool.

Thanks, I deleted this line.